contrib.json

```diff
@@ -1681,6 +1681,17 @@
                     "tags": true
                 }
             ]
+        },
+        {
+            "name": "SublimeLinter-makefile",
```

Comment:
3rd party linter adapters/plugins  need the `-contrib-` in their name.  That's a convention.

Reply:
Should I also change the (repo) name of my package?

Reply:
technically not necessary (maybe less confusing)

Reply:
OK, I added the `-contrib-` prefix and also renamed my repo for consistency.